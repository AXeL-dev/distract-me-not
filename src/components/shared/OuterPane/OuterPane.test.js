import { render } from "@testing-library/react";
import { OuterPane } from ".";

it('renders correctly', () => {
  const { asFragment } = render(
    <OuterPane>
      <span>Children</span>
    </OuterPane>
  );
  expect(asFragment()).toMatchSnapshot();
});
