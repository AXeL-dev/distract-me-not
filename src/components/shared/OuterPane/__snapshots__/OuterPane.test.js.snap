// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="css-nil ub-box-szg_border-box"
  >
    <span>
      Children
    </span>
  </div>
</DocumentFragment>
`;
